src/usr/lib/rsetup/tui/task/baota/baota.sh

```diff
@@ -0,0 +1,31 @@
+# shellcheck shell=bash
+
+__task_baota_uninstall() {
+    if yesno "Are you sure to uninstall Baota?"
+    then
+        if uninstall_baota
+        then
+            msgbox "Baota was successfully uninstalled."
+        else
+            msgbox "Failed to uninstall Baota." "$RTUI_PALETTE_ERROR"
+        fi
+    fi
+}
+
+__task_baota_install() {
+    if yesno "Are you sure you want to install Baota?"
+    then
+        install_baota
+    fi
+}
+
+__task_baota() {
+    menu_init
+    if [ -f /www/server/panel/BT-Panel ]
```

Comment:
```suggestion
    if [[ -f /www/server/panel/BT-Panel ]]
```
`[ ]` should [generally be avoided](https://unix.stackexchange.com/questions/67172/vs-which-one-to-use-in-bash-scripts).